runtime/lua/vim/lsp/util.lua

```diff
@@ -572,6 +573,12 @@ do
     api.nvim_buf_clear_namespace(bufnr, diagnostic_ns, 0, -1)
   end
 
+  function M.buf_clear_references(bufnr)
+    validate { bufnr = {bufnr, 'n', true} }
+    bufnr = bufnr == 0 and api.nvim_get_current_buf() or bufnr
```

Comment:
`nvim_buf_clear_namespace` already handles `0` so  this shouldn't be needed (here or line 572)

Reply:
Will fix, thanks